style/js/shield_defs.js

```diff
@@ -439,6 +439,99 @@ export function loadShields(shieldImages) {
   shields["US:DE:Business"] = banneredShield(shields["US:DE"], ["BUS"]);
   shields["US:DE:Truck"] = banneredShield(shields["US:DE"], ["TRK"]);
 
+  shields["US:FL"] = {
+    backgroundImage: [
+      shieldImages.shield40_us_fl_2,
+      shieldImages.shield40_us_fl_3,
+    ],
+    textColor: "black",
+    padding: {
+      left: 2,
+      right: 4.5,
+      top: 6,
+      bottom: 4,
+    },
+  };
+  shields["US:FL:Toll"] = {
+    backgroundImage: shieldImages.shield40_us_fl_toll,
+    textColor: "black",
+    padding: {
+      left: 2,
+      right: 5,
+      top: 7.5,
+      bottom: 4.5,
+    },
+  };
+  [
+    "Alachua",
+    "Baker",
+    "Bay",
+    "Bradford",
+    "Brevard",
+    "Broward",
+    "Calhoun",
+    "Charlotte",
+    "Citrus",
+    "Clay",
+    "Collier",
+    "Columbia",
+    "DeSoto",
+    "Dixie",
+    "Duval",
+    "Escambia",
+    "Flagler",
+    "Franklin",
+    "Gadsden",
+    "Gilchrist",
+    "Glades",
+    "Gulf",
+    "Hamilton",
+    "Hardee",
+    "Hendry",
+    "Hernando",
+    "Highlands",
+    "Hillsborough",
+    "Holmes",
+    "Indian River",
+    "Jackson",
+    "Jefferson",
+    "Lafayette",
+    "Lake",
+    "Lee",
+    "Leon",
+    "Levy",
+    "Liberty",
+    "Madison",
+    "Manatee",
+    "Marion",
+    "Martin",
+    "Miami-Dade",
```

Comment:
[Wikipedia says](https://en.wikipedia.org/wiki/Miami-Dade_County,_Florida#County_roads) none of Miami-Dade County’s county roads have signposted route numbers.